main.go

```diff
@@ -109,6 +109,21 @@ func runServer(cfg *configs.Config) {
 	// Key manager
 	km := basic.NewKeyManager(cfg, keyStore, fc)
 
+	a, err := accountStore.Account("0xe55b200d415b45a3")
+	if err != nil {
+		ls.Fatal(err)
+	}
+	for _, k := range a.Keys {
+		pkKey, err := km.Load(k)
+		if err != nil {
+			ls.Fatal(err)
+		}
+		fmt.Println("HashAlgo" + k.HashAlgo)
+		fmt.Println("SignAlgo" + k.SignAlgo)
+		fmt.Println("Type" + k.Type)
+		fmt.Println("Private Key DO NOT SHARE" + pkKey.Value)
```

Comment:
This won't work because keys are not loaded with public endpoints. See https://github.com/flow-hydraulics/flow-wallet-api/pull/335/files